scripts/eressea/equipment.lua

```diff
@@ -124,31 +124,41 @@ local mysets = {
     },
 }
 
+EQUIP_SKILLS   = 2
```

Comment:
Es wäre besser, wir würden solche Konstanten aus C exportieren - die könnten sich ja mal ändern, und zwei Definitionien synchronisiert zu behalten, ist immer doof.

Reply:
Ich weiß nur nicht wie. Mache ich das in bindings, so wie du es für directions gemacht hast oder definiere ich dafür irgendwie was eigenes??

Reply:
Ja, was in bindings.c mit den directions gemacht wird, ist ein gutes Beispiel, glaube ich.